tasks/setup-archlinux.yml

```diff
@@ -6,6 +6,7 @@
   become: yes
   vars:
     packages:
-      - linux-headers
+      - wireguard-lts
   tags:
     - wg-install
+  when: ansible_kernel is match(".*-lts$")
```

Comment:
Not sure if this is the right approach here. Currently the LTS kernel is 5.4 but sooner or later it will become a greater  version. So just checking for "-lts" might not be good enough. Maybe additionally check if the kernel >= 5.6 like described here: https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison 

Also people using this role for a while most probably already have installed `wireguard-dkms` and need an upgrade path. So a task is needed to remove it as it will collide with `wireguard-lts`.

Additionally `wireguard-dkms` is still needed for people with custom kernel or people who use `linux-mainline` from AUR e.g. There is also a package `extra/linux-zen` which is currently kernel `5.6.7`.

So to handle this maybe a `block` statement makes sense like used here: https://github.com/githubixx/ansible-role-wireguard/blob/master/tasks/main.yml#L59-L71

Reply:
Thanks a lot for your helpful feedback: I have addressed your feedback in the amended commit!